src/components/Contact.js

```diff
@@ -58,9 +58,18 @@ class Contact extends React.Component {
     );
   }
 
-  // eslint-disable-next-line class-methods-use-this
   createFavouriteContact() {
```

Comment:
You can merge createNormalContact and createFavouriteContact into one function and only show/hide content by having a isFavourite bool. 

`{ !isFavourite && (<div className="column"> <Avatar image={contact.image} /> </div>) }`
In this example, the avatar image will only be displayed if the contact is NOT favourite, so use this in case you need things only to be displayed for the normal contact.

Another thing to mention here would be to use the following naming convention when you have functions that return JSX code: `renderSomething() {}`. Use the render keyword at the beginning.